style.css

```diff
@@ -21,3 +21,9 @@
   opacity: 1;
   transition: opacity 0.2s;
 }
+.img-wrapper {
+  width: 40vw;
+}
+.img-responsive {
+  height: 200px;
```

Comment:
Not sure why this height property is mentioned here. if this is for documentation purpose I would suggest using a different class name for height. img-responsive class is already mentioned at the top of the file.